tests/libdnf/sack/QueryTest.cpp

```diff
@@ -57,23 +66,28 @@ void QueryTest::testQueryGetAdvisoryPkgs()
     CPPUNIT_ASSERT(!g_strcmp0(advisoryPkgs[0].getNameString(), "test-perl-DBI"));
     CPPUNIT_ASSERT(!g_strcmp0(advisoryPkgs[1].getNameString(), "test-perl-DBI"));
 
-    dnf_sack_filter_modules_v2(sack, nullptr, nullptr, tmpdir, nullptr, false, false, false);
+    // When module are setup but none are enabled all collections are not applicable - no enabled module
```

Comment:
This is actually my mistake, but can you please fix it? It is copied in a couple of other places too.
`module` -> `modules`